docs/source/alfacase/03_alfacase_example.rst

```diff
@@ -0,0 +1,121 @@
+.. _alfacase-example:
+
+ALFAcase Example
+================
+
+In this section, we will create a project from scratch with a simple network and insert it on the ALFAsim application.
```

Comment:
I don't think this page (the file) adds much: we should guide the user to produce a case file using the application at least as a starting point, and then build from there using the Case documentation as guide.

Reply:
> We should guide the user to produce a case file using the application at least as a starting point

Hmm, I don't know if I like this idea, the `ALFAcase` generated from the application has all values filled and it brings too much information for something that should guide the user.

I think that I will split this example into sections, introducing the options and after that instructing how to fill the separately the PVT,  Output, Pipe, and Node entries. 
With this, I think that the user will be ready to follow the API reference and edit a full alfacase generated from the application.

What did you think about this?

Reply:
TBH I don't think users will be generating files from scratch, rather they will always build up a small project, generate a case, and edit things from there. But go with your gut feeling then.

Reply:
Hmm I'm not sure which will be the common workflow too, but I also feel like creating it from scratch doesn't seem like the way to go (our only user so far is PWPa and they didn't go that path, right?). I can think of three more plausible ways: generating a project and editing it, copying an example or using a template.

But TBH I don't think we should overthink this, it will be a long time before we have real users of this feature, so do as you wish.

Reply:
Commit b3e9df5 improves the example section.